tests/decoder/factors/test_factors.sh

```diff
@@ -0,0 +1,22 @@
+#/bin/bash -x
+
+#####################################################################
+# SUMMARY: Tests decoding with factors
+# AUTHOR: pedrodiascoelho
+# TAGS: factors
+#####################################################################
+
+# Exit on error
+set -e
+
+# Remove old artifacts
+rm -f factors.{out,diff}
+
+# Run marian decoder
+$MRT_MARIAN/marian-decoder -c $MRT_MODELS/factors/model.npz.decoder.yml < text.in > factors.out
+
+# Compare the output with the expected output
```

Comment:
Can we also check if factors are loaded/used? For example for looking for a specific message in marian logs if there is one?

Reply:
Added check for "Factored embeddings enabled" and "Factored outputs enabled".

Reply:
Thanks!